araig_gui/src/araig_gui/araig_gui.cpp

```diff
@@ -90,18 +79,18 @@ void MyPlugin::spawnPubs()
   }
 }
 
-void MyPlugin::spawnSubs()
+void AraigGui::spawnSubs()
 {
   for(int i=0; i<num_outputs_; i++)
   {
     std::string topicName = "/signal/runner/" + name_outputs_[static_cast<std::vector<std::string>::size_type>(i)];
```

Comment:
When part of a bigger project, it is important to not start the topic names with "/" because this will put the topic always under the "/" namespace and not allow for namespace separation (for example by using the ns tag in launch). In AMBS specifically, all nodes are spawned under the "AMBS" namespace in the launch files, adding a clear separation from all other robot/sensor/etc.. topics. So please remove the first "/" from all topic names.

Reply:
The "/" in topic is to make the topics correspond the topics of runner, for remap function not available for GUI.
I use the static cast only because Qt creator always raises warnings on it if I don't. It's annoying, so I changed them. Basically, I don't think I need them.